packages/veritone-react-common/src/components/StatusPill/index.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { string } from 'prop-types';
+import styles from './styles.scss';
+
+const stateStyles = {
```

Comment:
I believe you can just use the `Lozenge` component for this instead of creating a new component. If that's not the case then this needs a story attached to it. Also you should import the colors from material-ui, i.e - `import purple from '@material-ui/core/colors/purple';`